js_modules/dagit/packages/core/src/scripts/generateGraphQLTypes.ts

```diff
@@ -39,8 +41,10 @@ writeFileSync('./src/graphql/possibleTypes.generated.json', JSON.stringify(possi
 console.log('Generating TypeScript types...');
 
 execSync(
-  'find src -type d -name types | xargs rm -r && yarn apollo codegen:generate --includes "./src/**/*.tsx" --target typescript types --localSchemaFile ./src/graphql/schema.graphql --globalTypesFile ./src/types/globalTypes.ts',
+  `find src -type d -name types | xargs rm -r && yarn apollo codegen:generate --includes "./src/**/*.tsx" --target typescript types --localSchemaFile ${TARGET_FILE} --globalTypesFile ./src/types/globalTypes.ts`,
   {stdio: 'inherit'},
 );
 
+execSync(`prettier --loglevel silent --write ${TARGET_FILE}`, {stdio: 'inherit'});
```

Comment:
this might run a global `prettier` which isn't necessarily the right version right?

Reply:
Hmm interesting, maybe we do an explicit ../../node_modules/.bin/prettier?

Reply:
Actually since this is JS I wonder if we could leverage the node package resolution and use the prettier JS api like this: https://prettier.io/docs/en/api.html

Only downside is it seems we'd have to read + write the file ourselves...

Reply:
otherwise exec-ing the right `yarn` command to run prettier should work assuming the current working directory is right

Reply:
This is correct, it is now `yarn prettier`.